spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/adapter/MessageListenerAdapter.java

```diff
@@ -339,11 +339,34 @@ protected String getListenerMethodName(Message originalMessage, Object extracted
 	 * @param extractedMessage the content of the message
 	 * @return the array of arguments to be passed into the listener method (each element of the array corresponding to
 	 * a distinct method argument)
+	 * @deprecated use @{org.springframework.amqp.rabbit.listener.adapter.MessageListenerAdapter#buildListenerArguments
```

Comment:
`{@link`, can be without classname, only a method reference starting with `#`

Reply:
thx